README.md

```diff
@@ -49,7 +48,8 @@ versions of this package used "M-p"
 
 In order to get the the client ID and client secret, you need to create
 [a Spotify app](https://developer.spotify.com/my-applications), specifying
-<http://localhost:8591/> as the redirect URI.
+<http://localhost:8080/spotify-callback> as the redirect URI (or whichever port you have specified via customize).
```

Comment:
If we decide to change the default port to `8591`, we'll need to change this as well.

```suggestion
<http://localhost:8591/spotify-callback> as the redirect URI (or whichever port you have specified via customize).
```